frontend/src/lib/k8s/api/v1/portForward.ts

```diff
@@ -74,14 +74,18 @@ export async function startPortForward(
   id: string = ''
 ): Promise<PortForward> {
   const kubeconfig = await findKubeconfigByClusterName(cluster);
-  const headers: HeadersInit = {
-    Authorization: `Bearer ${getToken(cluster)}`,
+  const headers: HeadersInit = new Headers({
```

Comment:
new Headers() is not really needed, just plain object works fine
but it doesn't really matter